---
title: "Nunca se alcanz&#243; el bloque &#39;Catch&#39; porque &#39;&lt;nombre1&gt;&#39; hereda de &#39;&lt;nombre2&gt;&#39; | Microsoft Docs"
ms.custom: ""
ms.date: "11/16/2016"
ms.prod: "visual-studio-dev14"
ms.reviewer: ""
ms.suite: ""
ms.technology: 
  - "devlang-visual-basic"
ms.tgt_pltfrm: ""
ms.topic: "article"
f1_keywords: 
  - "BC42029"
  - "vbc42029"
helpviewer_keywords: 
  - "BC42029"
ms.assetid: 4bc84a84-ec7e-49db-8794-4fb71d36d5c0
caps.latest.revision: 12
caps.handback.revision: 12
author: "stevehoag"
ms.author: "shoag"
manager: "wpickett"
---
# Nunca se alcanz&#243; el bloque &#39;Catch&#39; porque &#39;&lt;nombre1&gt;&#39; hereda de &#39;&lt;nombre2&gt;&#39;
Un bloque `Catch` dentro del código nunca se alcanza.  
  
 De forma predeterminada, este mensaje es una advertencia. Para obtener más información sobre cómo ocultar las advertencias o cómo tratarlas como errores, vea [Configurar advertencias en Visual Basic](../Topic/Configuring%20Warnings%20in%20Visual%20Basic.md).  
  
 **Identificador de error:** BC42029  
  
### Para corregir este error  
  
-   Elimine el bloque `Catch` superfluo.  
  
## Vea también  
 [Cómo: Probar el código con un bloque Try...Catch en Visual Basic](http://msdn.microsoft.com/es-es/8368e205-ed73-4185-a247-af84fb4fafa9)   
 [Try...Catch...Finally \(Instrucción\)](../Topic/Try...Catch...Finally%20Statement%20\(Visual%20Basic\).md)